data/hai/hai outfits.txt

```diff
@@ -11,6 +11,38 @@
 # You should have received a copy of the GNU General Public License along with
 # this program. If not, see <https://www.gnu.org/licenses/>.
 
+outfit "Phantom Pallet"
+	category "Systems"
+	cost 12000
+	thumbnail "outfit/phantom pallet"
+	"mass" 0.2
+	"unplunderable" 1
+	"energy consumption" 0.016667
+	"heat generation" 0.03334
+
+	# Replacing 30% of your cargo bay with these triples your cargo scan time.
+	"cargo space" -1
+	"cargo scan opacity" 15
+
+	description "This Unfettered modification of the standard Hai Holopad is a borderline-illegal smuggling tool."
+	description "Holopads are small versions of the Hai Holovid Zone. They have holographic projectors, textured forcefields, and phase array speakers. As a mid-priced but powerful option, it's often seen in meeting rooms, arcades, and personal gaming setups. Several can be packed together to make a larger holographic space."
+	description "Unfettered Phantom Pallets are more durable and stealthy Holopads. They can hide under floors, in walls, or in ceilings, and remain undetected. Install them in your cargo bay and project fake cargo as sensor ghosts. They confuse scanners into scanning empty space instead of contraband. That can make the difference between seized cargo and delivering needed goods."
```

Comment:
```suggestion
	description "Unfettered Phantom Pallets are more durable and stealthy Holopads. They can hide under floors, in walls, or in ceilings while remaining undetected. Install them in your cargo bay and project fake cargo as sensor ghosts. They confuse scanners into scanning empty space instead of contraband. That can make the difference between seized cargo and delivering needed goods."
```
I think this flows better, but it's up to you